DECISIONLOG.md

```diff
@@ -0,0 +1,3 @@
+# Snapchat Ads Decision Log
```

Comment:
We should also add the UTM filtering details here as well. See the decisionlog entry for [Google Ads](https://github.com/fivetran/dbt_google_ads/blob/feature/v2_updates/DECISIONLOG.md#utm-report-filtering) for format and reasoning.

> The UTM filtering comment is in a comment attached to the utm report model.